### ISSUE TITLE
Add new beluga release repository

### ISSUE THREAD

**Issue by hidmic** [ros2-gbp/ros2-gbp-github-org#520]
* Name of the release team: [`beluga`](https://github.com/ros2-gbp/ros2-gbp-github-org/issues/518)

* Release repositories to add:
  
  * [Ekumen-OS/beluga](https://github.com/Ekumen-OS/beluga)
    * [noetic source entry](https://github.com/ros/rosdistro/blob/33229c39fb48111a754c18c67a7e85e558fad9a7/noetic/distribution.yaml#L572-L581)
    * [humble source entry](https://github.com/ros/rosdistro/blob/33229c39fb48111a754c18c67a7e85e558fad9a7/humble/distribution.yaml#L795-L804)
    * [iron source entry](https://github.com/ros/rosdistro/blob/33229c39fb48111a754c18c67a7e85e558fad9a7/iron/distribution.yaml#L617-L626)
    * [jazzy source entry](https://github.com/ros/rosdistro/blob/33229c39fb48111a754c18c67a7e85e558fad9a7/jazzy/distribution.yaml#L655-L664)

**Comment by hidmic** [comment]
Depends on #518. Proactively making this request so that both release team and repository can be processed in one go. HTH.

**Comment by nuclearsandwich** [comment]
Thanks @hidmic. For future reference we can add release repositories in the new team issue as well. There's no need for a separate issue unless you're adding new repositories to an existing team.

**Comment by nuclearsandwich** [comment]
Resolved by #524 but the auto-closer doesn't understand "and".
